packages/analytics/analytics-chart/src/composables/useChartLegendValues.ts

```diff
@@ -13,13 +12,9 @@ export default function useChartLegendValues(chartData: Ref<KChartData>, chartTy
     return chartData.value.datasets.reduce((a, v) => {
       const raw = v.total
         ? v.total
-        : [ChartTypes.TIMESERIES_BAR, ChartTypes.TIMESERIES_LINE].some(e => e === chartType)
-          ? (v.data as ScatterDataPoint[]).reduce((acc: number, { y }: ScatterDataPoint) => {
-            return acc + (Number(y) || 0)
-          }, 0) as number
-          : (v.data as DoughnutDataPoint[]).reduce((acc: number, e: number) => {
-            return acc + Number(e)
-          }, 0) as number
+        : (v.data as Array<number | AnalyticsDataPoint>).reduce((acc: number, e: number | AnalyticsDataPoint) => {
+          return acc + (typeof e === 'number' ? Number(e) : e.y)
```

Comment:
Ahh good catch, that's still needed, in case `e.y` is not a number

<img width="207" alt="image" src="https://github.com/Kong/public-ui-components/assets/6026470/3aaddba7-c079-4bbf-8125-4454daef58b6">